packages/admin/src/Resources/Pages/EditRecord.php

```diff
@@ -161,15 +165,8 @@ protected function getViewAction(): Action
 
     protected function getDeleteAction(): Action
```

Comment:
Since these are all single line functions then, we could deprecate them now and remove them with V3.

Reply:
yup 👍